config/controllers/hive_controllers_role.yaml

```diff
@@ -38,10 +38,18 @@ rules:
   - update
   - patch
   - delete
+- apiGroups:
```

Comment:
The controllers should not need delete rights to the pods directly. All of the deletes should be to jobs instead of pods.